src/patternfly/utilities/Text/text.scss

```diff
@@ -0,0 +1,47 @@
+// stylelint-disable
+
+// Font size options
+$pf-u-font-size-options: (
+  font-size-xs:  (font-size var(--pf-global--FontSize--xs)),
+  font-size-sm:  (font-size var(--pf-global--FontSize--sm)),
+  font-size-md:  (font-size var(--pf-global--FontSize--md)),
+  font-size-lg:  (font-size var(--pf-global--FontSize--lg)),
+  font-size-xl:  (font-size var(--pf-global--FontSize--xl)),
+  font-size-2xl: (font-size var(--pf-global--FontSize--2xl)),
+  font-size-3xl: (font-size var(--pf-global--FontSize--3xl)),
+  font-size-4xl: (font-size var(--pf-global--FontSize--4xl))
+);
+
+@include pf-utility-builder($pf-u-font-size-options, $pf-global--breakpoint-list);
+
+
+// Font weight options
+$pf-u-font-weight-options: (
+  font-weight-light:              (font-weight var(--pf-global--FontWeight--light)),
+  font-weight-normal:             (font-weight var(--pf-global--FontWeight--normal)),
+  font-weight-bold:               (font-weight var(--pf-global--FontWeight--bold)),
+);
+
+@include pf-utility-builder($pf-u-font-weight-options, $pf-global--breakpoint-list);
+
+
+// Font color options
+$pf-u-font-color-options: (
+  color-100: (color var(--pf-global--Color--100)),
+  color-200: (color var(--pf-global--Color--200)),
+  color-300: (color var(--pf-global--Color--300)),
+  color-400: (color var(--pf-global--Color--400)),
+);
+
+@include pf-utility-builder($pf-u-font-color-options, $pf-global--breakpoint-list);
+
+// Font layout options
+$pf-u-font-alignment-options: (
+  font-break-word: (word-break break-word),
+  font-nowrap:     (white-space nowrap),
+  font-wrap:       (white-space normal),
+);
+
+@include pf-utility-builder($pf-u-font-alignment-options, $pf-global--breakpoint-list);
+
```

Comment:
I'd like to see truncate added, it would look a little different.. I don't know how I feel about making all of the values `!important`.. @christiemolloy @mcoker wdyt?

```

.pf-u-text-truncate {
  min-width: 0 !important;
  max-width: 100% !important;
  overflow: hidden !important;
  text-overflow: ellipsis !important;
  white-space: nowrap !important;
  display: inline-block !important;
}
```

Reply:
I think we would add at least what's in our mixin - white-space, overflow, text-overflow. You need those at a minimum, and that should handle most cases. I wouldn't add display though since we can't predictably change that, so I think it can be up to the user to make sure their element is block, kind of like needing a block element with something like a block/vertical margin.

Though I would add `!important` since utilities are designed to be added to anything and we never know what we're styling. They're basically like writing an inline style with a class. What's your hesitation?

Reply:
> Though I would add !important since utilities are designed to be added to anything and we never know what we're styling. They're basically like writing an inline style with a class. What's your hesitation?

Good point, 🚢  it!